src/pyOpenMS/pxds/MassTraceDetection.pxd

```diff
@@ -19,6 +20,7 @@ cdef extern from "<OpenMS/FILTERING/DATAREDUCTION/MassTraceDetection.h>" namespa
         MassTraceDetection()      nogil except +
 
         void run(MSExperiment & input_map,
-                libcpp_vector[Kernel_MassTrace] & traces
+                libcpp_vector[Kernel_MassTrace] & traces,
+                Size max_traces
```

Comment:
the C++ API is currently 
https://github.com/OpenMS/OpenMS/blob/57eb6a91ec9bed82a54712daec0380ec00583abe/src/openms/include/OpenMS/FILTERING/DATAREDUCTION/MassTraceDetection.h#L93
so this wont work - is the `max_traces` in some PR?

Reply:
The PR is against the proteomicslfq branch. 
https://github.com/OpenMS/OpenMS/blob/feature/proteomic_lfq/src/openms/include/OpenMS/FILTERING/DATAREDUCTION/MassTraceDetection.h

I've already added max_traces in the c++ part on that branch.